auth/README.md

```diff
@@ -2,6 +2,8 @@
 
 Go client library for signing in user.
```

Comment:
Can it  be improved as `Go client library for signing in sure with apple`?

Reply:
i think i mistakenly typed sure, it's not meaningful here.

Reply:
`Sure` can have meaning here. But for simplicity let's remove it

Reply:
sorry, but i didn't get it's mean. what's it?

Reply:
Signing in with apple without any doubt